src/Spotify/Library.java

```diff
@@ -0,0 +1,73 @@
+package Spotify;
+import java.io.*;
+import java.util.ArrayList;
+import Spotify.Library;
+import Spotify.Album;
+import Spotify.Playlist;
+
+public class Library {
+
+  // instance variables
+  private String name;
+  private ArrayList <Playlist> Library = new ArrayList<Playlist>();
```

Comment:
'Library' is confusing as an instance variable name since it is the same name as the class and starts with an upper case letter (it shouldn't)